FreeRTOS/iot-reference-arm-corstone3xx
# build-run: Add Ethos NPU ID and MAC configuration option

Add Ethos NPU ID and MAC configuration option

Description
-----------
- Selectable U55/U65 NPU
- Selectable MACs for the NPU
- Update linker files to support Dedicated SRAM mode for Ethos-U65

Test Steps
-----------
Existing tests were run, U65 option checked for Speech recognition and KWS examples.

Checklist:
----------



Related Issue
-----------

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
